recipes/vsgimgui/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import get, copy, rm, rmdir, collect_libs, apply_conandata_patches, export_conandata_patches
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake
+import os
+
+required_conan_version = ">=1.53.0"
+
+class VsgImGuiConan(ConanFile):
+    name = "vsgimgui"
+    description = "VulkanSceneGraph ImGui integration"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.vulkanscenegraph.org"
+    topics = ("vulkan", "scenegraph", "graphics", "3d", "ImGui")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True, 
+    }
+    #to keep backends for imgui
+    keep_imports = True
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "7",
+            "apple-clang": "10",
+        }
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+       
+    def requirements(self):
+        self.requires("vulkan-loader/1.3.239.0", transitive_headers=True)
+        self.requires("vsg/[>1.0.5]")
```

Comment:
Hi @psyinf, thanks a lot for the contribution. 
Please, note that we are only accepting ranges for openssl and cmake for the moment: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges
Also, does this need >1.0.5? There's only until 1.0.5 i Conan Center
```suggestion
        self.requires("vsg/1.0.5")
```